server/models/Collection.ts

```diff
@@ -48,13 +48,23 @@ import NotContainsUrl from "./validators/NotContainsUrl";
   withAllMemberships: {
     include: [
       {
-        model: CollectionUser,
+        model: UserPermission,
         as: "memberships",
+        where: {
```

Comment:
Looking back at this now as part of the document permissions PR, I don't know why all these `[Op.ne]: null` statements were added. It doesn't really seem to achieve anything as we're already joining against the collection itself